app/models/legislation/process/phase.rb

```diff
@@ -10,7 +10,7 @@ def enabled?
   end
 
   def started?
-    enabled? && Date.current >= @start_date
+    return enabled? && Date.current >= @start_date unless @start_date.nil?
```

Comment:
`return` is unnecessary in this case, ruby has an implicit return. And regarding the use of `unless` for the nil check, a simple `if` would make the code more readable, like this:
```suggestion
    enabled? && Date.current >= @start_date if @start_date
```

Reply:
Fixed.